simulariumio/medyan/medyan_converter.py

```diff
@@ -103,13 +108,13 @@ def _parse_data_dimensions(
                 if "FILAMENT" in line:
                     # start of filament
                     cols = line.split()
-                    subpoints = int(cols[3])
+                    subpoints = 3 * int(cols[3])
                     if result.max_subpoints < subpoints:
                         result.max_subpoints = subpoints
                 else:
                     # start of linker or motor
-                    if result.max_subpoints < 2:
-                        result.max_subpoints = 2
+                    if result.max_subpoints < 6:
```

Comment:
I currently don't understand what's happening here, but even once I do understand, this seems like an if statement that could use an explanatory comment

Reply:
I added symbolic constants and more comments